packages/smooth_app/lib/database/contributor_product_query.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:smooth_app/data_models/product_list.dart';
+import 'package:smooth_app/database/paged_user_product_query.dart';
+
+/// User Product Search, in "contributor" mode.
+class ContributorProductQuery extends PagedUserProductQuery {
+  ContributorProductQuery(final String userId) : super(userId);
+
+  @override
+  ProductList getProductList() => ProductList.contributor(
+        userId,
+        pageSize: pageSize,
+        pageNumber: pageNumber,
+      );
+
+  @override
+  String getPath() => 'contributor/$userId.json';
+
+  @override
+  String toString() =>
+      'ContributorProductQuery("$userId", $pageSize, $pageNumber)';
```

Comment:
```suggestion
      'ContributorProductQuery($userId, $pageSize, $pageNumber)';
```
Are these quotes wanted
Same in the other queries

Reply:
I don't know how wanted they are, here we're talking about a `toString()` that displays a `String` between quotes.
Like `ContributorProductQuery("monsieurtanuki", 25, 1)`
And you suggest `ContributorProductQuery(monsieurtanuki, 25, 1)`
Let's say that I could live with both versions.